core/src/main/scala/org/apache/spark/network/BlockDataManager.scala

```diff
@@ -30,6 +30,13 @@ trait BlockDataManager {
   def getBlockData(blockId: BlockId): ManagedBuffer
 
   /**
+   * Interface to get the shuffle block data that block manager with given blockManagerId
+   * holds in a local host. Throws an exception if the block cannot be found or
+   * cannot be read successfully.
+   */
+  def getShuffleBlockData(blockId: ShuffleBlockId, blockManagerId: BlockManagerId): ManagedBuffer
```

Comment:
Renamed `getShuffleBlockData` into `getBlockData`. Is this fix correct for your comments?